Tests/TestRunners/ConverterTestBase.cs

```diff
@@ -28,6 +29,7 @@ public class ConverterTestBase
 
     public ConverterTestBase(string rootNamespace = null)
     {
+        CultureInfo.DefaultThreadCurrentUICulture = CultureInfo.InvariantCulture;
```

Comment:
Nice to have: Haven't had need to try this yet, but may be worth trying putting in a [ModuleInitializer ](https://www.cazzulino.com/module-initializers.html) so it definitely covers the whole test suite rather than some tests running with it and some without.